scripts/dts/extract_dts_includes.py

```diff
@@ -583,36 +584,49 @@ def yaml_traverse_inherited(node):
     ``inherits`` section is searched for and used as node base when found.
     Base values are then overloaded by node values
     Additionally, 'id' key of 'inherited' dict is converted to 'node_type'
+    and some consistency checks are done.
     :param node:
     :return: node
     """
 
+    # do some consistency checks. Especially id is needed for further
+    # processing. title must be first to check.
+    for prop in ('title', 'id', 'version', 'description'):
+        if prop not in node:
+            node[prop] = "<unknown {}>".format(prop)
+            print("extract_dts_includes.py: '{}' property missing".format(prop),
+                  "in '{}' binding. Using '{}'.".format(node['title'], node[prop]))
```

Comment:
If 'title' is not present, message is not much explicit and one can spend some time finding the faulty node.
What about using 'node' in warning message when title is not present?
Generated message won't be quite nice, but it will help finding the issue and the ugliness of the message could be an incentive to fix it...

Otherwise this check is a nice addition

Reply:
Added hint for no 'title' case.